### PR TITLE
update function create, update, list, edit  for Project, Member, Memb…

### PR DESCRIPTION
**Pull request by lethu96**
## Trello ticket URL
- create, update, delete,list Member
- create, update, delete,list Project
- create, update, delete,list MemberProject
-clean code
- Validate
## Technical changes overview
-none

## SQL update
- none

## Other issues
- noe

### CODE REVIEW

File: app/Http/Controllers/MemberController.php [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+use App\Position;
+use Validator;
+use DB;
+
+class MemberController extends Controller
+{
+    public function listMember(Request $request)
+    {
+        $listMember= Member::all()->toArray();
+        dd($listMember) ;
```
Comment (TaHieu-scuti) [inline app/Http/Controllers/MemberController.php]: Pls remove this line!

File: app/Http/Controllers/MemberController.php [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+    public function addMember(StoreCreateMember $request)
+    {
+        $data=$request->all();
+        $newMember=new Member();
+        $newMember->name=$data['name'];
+        $newMember->phone_number=$data['phone_number'];
+        $newMember->information=$data['information'];
+        $newMember->birthday=$data['birthday'];
+        $newMember->position_id=$data['position_id'];
+        $newMember->position =DB::table('positions')
```
Comment (TaHieu-scuti) [inline app/Http/Controllers/MemberController.php]: We just need Position_id field dont need position field. Pls remove Position field.

File: app/Http/Controllers/MemberController.php [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+    public function editMember(StoreCreateMember $request)
+    {
+        $data=$request->all();
+        $memberEdit=Member::find($data['id']);
+        $memberEdit->name=$data['name'];
+        $memberEdit->phone_number=$data['phone_number'];
+        $memberEdit->information=$data['information'];
+        $memberEdit->birthday=$data['birthday'];
+        $memberEdit->position_id=$data['position_id'];
+        $memberEdit->position = DB::table('positions')
```
Comment (TaHieu-scuti) [inline app/Http/Controllers/MemberController.php]: We just need Position_id field dont need position field. Pls remove Position field.

File: app/Http/Controllers/MemberProjectController.php [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+        $newMp=new MemberProject();
+        $newMp->meber_id=$data['member_id'];
+        $newMp->project_id=$data['project_id'];
+        $newMp->role=$data['role'];
+        $newMp->save();
+        return $newMp;
+    }
+    public function getEditMemberProject($id)
+    {
+
```
Comment (TaHieu-scuti) [inline app/Http/Controllers/MemberProjectController.php]: Pls run phpcs fixer for this file.

File: app/Http/Controllers/MemberController.php [hunk 5]
```diff
@@ -71,6 +71,6 @@ public function addMember(StoreCreateMember $request)
             $newMember->avatar = $file->getClientOriginalName();
         }
         $newMember->save();
-        return $newMember;
+         return $newMember;
```
Comment (TaHieu-scuti) [inline app/Http/Controllers/MemberController.php]: Condition Wrong. Pls remove this space.

**Review by TaHieu-scuti — APPROVED** [review]

**Review by ngon12 — APPROVED** [review]
OK for me

**Pull request merged**